### PR TITLE
Feature/archetype 21

### PR DESCRIPTION
**Pull request by godanny86**
- updated project structure to align with AEM Project Archetype 21

### CODE REVIEW

**Comment by godanny86** [comment]
@davidjgonzalez don't expect you to review all 290 files but was hoping you could take a quick look at the POM files and the All module...

File: all/pom.xml [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+                            <groupId>com.adobe.cq</groupId>
+                            <artifactId>core.wcm.components.content</artifactId>
+                            <type>zip</type>
+                            <target>/apps/wknd-vendor-packages/application/install</target>
+                        </embedded>
+                        <embedded>
+                            <groupId>com.adobe.cq</groupId>
+                            <artifactId>core.wcm.components.config</artifactId>
+                            <type>zip</type>
+                            <target>/apps/wknd-vendor-packages/application/install</target>
```
Comment (davidjgonzalez) [inline all/pom.xml]:
```suggestion
                            <target>/apps/wknd-vendor-packages/content/install</target>
```

File: all/pom.xml [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+                        </embedded>
+                        <embedded>
```
Comment (davidjgonzalez) [inline all/pom.xml]: Remove this embed; I believe core.wcm.components.examples is a mixed content pacakge

File: all/pom.xml [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+            <groupId>com.adobe.cq</groupId>
+            <artifactId>core.wcm.components.content</artifactId>
+            <type>zip</type>
+        </dependency>
+        <dependency>
+            <groupId>com.adobe.cq</groupId>
+            <artifactId>core.wcm.components.config</artifactId>
+            <type>zip</type>
+        </dependency>
+        <dependency>
```
Comment (davidjgonzalez) [inline all/pom.xml]: Remove this to since you dont need to be deploying examples as part of this?

File: repository-structure/pom.xml [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+        Any roots in the Code packages of this project should have their parent enumerated in the Filters list below.
+    </description>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.jackrabbit</groupId>
+                <artifactId>filevault-package-maven-plugin</artifactId>
+                <extensions>true</extensions>
+                <configuration>
```
Comment (davidjgonzalez) [inline repository-structure/pom.xml]:
```suggestion
                <configuration>
                <properties>
                    <cloudManagerTarget>none</cloudManagerTarget>
                </properties>                
```

File: ui.apps/pom.xml [hunk 5]
```diff
@@ -56,26 +49,41 @@
                 <artifactId>filevault-package-maven-plugin</artifactId>
                 <extensions>true</extensions>
                 <configuration>
-                    <group>wknd</group>
+                    <group>com.adobe.aem.guides</group>
+                    <name>aem-guides-wknd.ui.apps</name>
+                    <packageType>application</packageType>
+                    <accessControlHandling>merge</accessControlHandling>
+                    <properties>
+                        <cloudManagerTarget>none</cloudManagerTarget>>
```
Comment (davidjgonzalez) [inline ui.apps/pom.xml]:
```suggestion
                        <cloudManagerTarget>none</cloudManagerTarget>
```

File: ui.content/pom.xml [hunk 6]
```diff
@@ -55,8 +48,17 @@
                 <artifactId>filevault-package-maven-plugin</artifactId>
                 <extensions>true</extensions>
                 <configuration>
-                    <acHandling>merge_preserve</acHandling>
-                    <group>wknd</group>
+                    <group>com.adobe.aem.guides</group>
+                    <name>aem-guides-wknd.ui.content</name>
+                    <packageType>content</packageType>
```
Comment (davidjgonzalez) [inline ui.content/pom.xml]:
```suggestion
                    <packageType>content</packageType>
                    <properties>
                         <cloudManagerTarget>none</cloudManagerTarget>
                     </properties>
```

File: all/pom.xml [hunk 7]
```diff
@@ -0,0 +1,203 @@
...
+                <groupId>com.day.jcr.vault</groupId>
+                <artifactId>content-package-maven-plugin</artifactId>
+                <extensions>true</extensions>
+                <configuration>
+                    <verbose>true</verbose>
+                    <failOnError>true</failOnError>
+                </configuration>
+            </plugin>
+            <plugin>
+                <artifactId>maven-clean-plugin</artifactId>
```
Comment (davidjgonzalez) [inline all/pom.xml]: <plugin>
                <artifactId>maven-clean-plugin</artifactId>
                <version>2.4.1</version>
                <executions>
                    <execution>
                        <id>auto-clean</id>
                        <phase>initialize</phase>
                        <goals>
                            <goal>clean</goal>
                        </goals>
                    </execution>
                </executions>
            </plugin>

File: all/pom.xml [hunk 8]
```diff
@@ -0,0 +1,203 @@
...
+                <version>2.4.1</version>
+                <executions>
+                    <execution>
+                        <id>auto-clean</id>
+                        <phase>install</phase>
```
Comment (davidjgonzalez) [inline all/pom.xml]:
```suggestion
                        <phase>initialize</phase>
```

File: all/pom.xml [hunk 9]
```diff
@@ -0,0 +1,203 @@
...
+                <artifactId>content-package-maven-plugin</artifactId>
+                <extensions>true</extensions>
+                <configuration>
+                    <verbose>true</verbose>
+                    <failOnError>true</failOnError>
+                </configuration>
+            </plugin>
+            <plugin>
+                <artifactId>maven-clean-plugin</artifactId>
+                <version>2.4.1</version>
```
Comment (davidjgonzalez) [inline all/pom.xml]:
```suggestion
```

**Pull request closed without merging**